ubuntu-headless-qt/include/01_prepare_ubuntu_base.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# --------------------------------------------------------------------------#
+# function ubuntu_base_prepare use to prepare basic binaries of ubuntu os.
```

Comment:
This description looks like a function description. Can you update it and move the old description to the correct location of the mentioned functions?